Sources/Components/IconCollection/IconCollectionView.swift

```diff
@@ -80,8 +80,8 @@ extension IconCollectionView: UICollectionViewDelegateFlowLayout {
             let width = max(collectionView.frame.width / CGFloat(viewModels.count), height * 2)
             return CGSize(width: width, height: height)
         } else {
-            let width = min(collectionView.frame.width, collectionView.frame.height)
```

Comment:
I thought it was a smart thing to do for supporting landscape on phones. But now I see it wasn't 🙈